src/main.tsx

```diff
@@ -11,35 +10,37 @@ import { tenantTheme, ThemeName } from './models/Theme';
 import { NodeInfoProvider } from './NodeInfoProvider';
 import SharedProvider from './SharedProvider';
 
-const queryClient = new QueryClient();
+import { StrictMode } from 'react';
+import { createRoot } from 'react-dom/client';
 
-/* const localStoragePersister = createSyncStoragePersister({ storage: window.localStorage });
-persistQueryClient({
-  queryClient,
-  persister: localStoragePersister,
-}); */
+const queryClient = new QueryClient();
 
-render(
-  <QueryClientProvider client={queryClient}>
-    <BrowserRouter>
-      <GlobalStateProvider>
-        <GlobalStateContext.Consumer>
-          {(globalState) => {
-            const { tenantRPC, getThemeName = emptyFn } = globalState as GlobalState;
-            return (
-              <NodeInfoProvider tenantRPC={tenantRPC}>
-                <SharedProvider>
-                  <Theme dataTheme={`${getThemeName()}`} className={tenantTheme[getThemeName() || ThemeName.Pendulum]}>
-                    <App />
-                  </Theme>
-                </SharedProvider>
-              </NodeInfoProvider>
-            );
-          }}
-        </GlobalStateContext.Consumer>
-      </GlobalStateProvider>
-    </BrowserRouter>
-    <ReactQueryDevtools initialIsOpen={false} position="bottom-right" />
-  </QueryClientProvider>,
-  document.getElementById('app') as HTMLElement,
+createRoot(document.getElementById('root')!).render(
+  <StrictMode>
+    <QueryClientProvider client={queryClient}>
+      <BrowserRouter>
+        <GlobalStateProvider>
+          <GlobalStateContext.Consumer>
+            {(globalState) => {
+              const { tenantRPC, getThemeName = emptyFn } = globalState as GlobalState;
+              return (
+                <NodeInfoProvider tenantRPC={tenantRPC}>
+                  <SharedProvider>
+                    <Theme
+                      dataTheme={`${getThemeName()}`}
+                      className={tenantTheme[getThemeName() || ThemeName.Pendulum]}
+                    >
+                      <App />
+                    </Theme>
+                  </SharedProvider>
+                </NodeInfoProvider>
+              );
+            }}
+          </GlobalStateContext.Consumer>
+        </GlobalStateProvider>
+      </BrowserRouter>
+      <ReactQueryDevtools initialIsOpen={false} position="bottom-right" />
+    </QueryClientProvider>
+    ,
```

Comment:
I suppose that this comma here is wrong.

Reply:
Thanks!